src/components/breadcrumbs/crumb/crumb.spec.tsx

```diff
@@ -0,0 +1,49 @@
+import React from "react";
+import { render, screen } from "@testing-library/react";
+import userEvent from "@testing-library/user-event";
+import { Crumb } from "./crumb.component";
+
+const LINK_TEXT = "Link text";
+
+describe("Crumb", () => {
+  it("passes href to the anchor element isCurrent is false", () => {
+    render(<Crumb href="foo" text={LINK_TEXT} />);
```

Comment:
If we extend the `LinkProps` we can just pass the text as children

```
<Crumb href="foo">{LINK_TEXT}</Crumb>
```

Reply:
In this case we could remove the `text` prop and use `children` instead 👍🏼